docs/learn/encyclopedia/transactions-specialized/channel-accounts.mdx

```diff
@@ -64,16 +64,16 @@ secret_key = "SDY5TRQSEUSHS7UX26QMNMZ4X543UWZQPJZ7LQQUA3NAKDFVYTAWAB74"
 source_keypair = Keypair.from_secret(secret_key)
 source_account = server.load_account(source_keypair.public_key)
 
-transaction_1 = TransactionBuilder(
+transaction1 = TransactionBuilder(
   source_account = source_account,
   network_passphrase = Network.TESTNET_NETWORK_PASSPHRASE,
-  base_fee = 100
+  base_fee = 640
 )
 
-transaction_2 = TransactionBuilder(
+transaction2 = TransactionBuilder(
   source_account = source_account,
   network_passphrase = Network.TESTNET_NETWORK_PASSPHRASE,
-  base_fee = 100
+  base_fee = 640
```

Comment:
Same as above. Add comment as to why youre suggesting 640 base fee is 100 outside of surge pricing